### DISCUSSION TITLE
Is Windows 7/older cpu's not supported?

### DISCUSSION THREAD

**Discussion by userse31** [icsharpcode/ILSpy#3148] [General]
I primarily use linux and a series of hard drive shenanigans means _I don't have a modern computer running Windows 10!_

Infact, I HATE modern Windows!

Installed Windows 7 on an old Athlon 64 machine for _Tracfone_ related reasons.

ILSpy immediately quits 15 fempto seconds after running.

I do know modern "amd64" cpus have added SO MANY EXTRAS that older processors are essentially EOL with stuff like modern games.

**Comment by greenozon** [comment]
Have you installed .NET framework(s) on your OS?
even .NET 8 is absolutely fine to install on great (the greatest!) OS = Windows 7
;)
